2024-09-26 15:16:36,390 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:21:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:21:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:21:34,227 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:21:34,965 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:21:36,267 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:26:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:26:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:26:34,182 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:26:35,057 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:26:36,310 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:31:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:31:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:31:34,113 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:31:34,847 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:31:36,073 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:36:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:36:33,465 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:36:34,162 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:36:35,019 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:36:36,221 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:41:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:41:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:41:34,177 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:41:34,984 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:41:36,218 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:46:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:46:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:46:34,167 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:46:34,902 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:46:36,177 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:51:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:51:33,462 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:51:34,181 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:51:34,904 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:51:37,364 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 15:56:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 15:56:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 15:56:34,186 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 15:56:35,014 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 15:56:36,222 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 16:01:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 16:01:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 16:01:34,093 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 16:01:35,167 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
2024-09-26 16:01:36,424 ERROR: Error in fetch_emails: (psycopg2.errors.UndefinedTable) relation "email_fetch_tracker" does not exist
LINE 2: FROM email_fetch_tracker 
             ^

[SQL: SELECT email_fetch_tracker.id AS email_fetch_tracker_id, email_fetch_tracker.last_fetch_time AS email_fetch_tracker_last_fetch_time 
FROM email_fetch_tracker 
 LIMIT %(param_1)s]
[parameters: {'param_1': 1}]
(Background on this error at: https://sqlalche.me/e/20/f405) [in /home/runner/CRM/email_receiver.py:207]
2024-09-26 16:06:33,443 INFO: Checking for new emails [in /home/runner/CRM/email_receiver.py:235]
2024-09-26 16:06:33,461 INFO: Attempting to connect to imap.gmail.com:993 with SSL version: 2 [in /home/runner/CRM/email_receiver.py:34]
2024-09-26 16:06:34,119 INFO: IMAP4_SSL connection established with SSL version: 2 [in /home/runner/CRM/email_receiver.py:36]
2024-09-26 16:06:34,878 INFO: Successfully logged in to the email server [in /home/runner/CRM/email_receiver.py:38]
